controllers/commonservice_controller.go

```diff
@@ -97,6 +97,16 @@ func (r *CommonServiceReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 		return ctrl.Result{}, client.IgnoreNotFound(err)
 	}
 
+	masterCR := &apiv3.CommonService{}
+	if err := r.Bootstrap.Client.Get(ctx, types.NamespacedName{Namespace: r.Bootstrap.CSData.OperatorNs, Name: "common-service"}, masterCR); err != nil {
+		return ctrl.Result{}, err
+	}
+
+	if !masterCR.Spec.License.Accept {
+		klog.Info("Accept license by changing .spec.license.accept to true in the CertManagerConfig CR. Operator will not proceed until then")
```

Comment:
Should this info line be referencing the commonservice CR instead of the CertManagerConfig CR?

Reply:
yes, good catch